.editorconfig

```diff
@@ -268,3 +268,16 @@ ij_xml_space_after_tag_name = false
 ij_xml_space_around_equals_in_attribute = false
 ij_xml_space_inside_empty_tag = false
 ij_xml_text_wrap = normal
+
+[{*.yaml,*.yml}]
+indent_size = 2
+ij_yaml_align_values_properties = do_not_align
+ij_yaml_autoinsert_sequence_marker = true
+ij_yaml_block_mapping_on_new_line = false
+ij_yaml_indent_sequence_value = true
+ij_yaml_keep_indents_on_empty_lines = false
+ij_yaml_keep_line_breaks = true
+ij_yaml_sequence_on_new_line = false
+ij_yaml_space_before_colon = false
+ij_yaml_spaces_within_braces = true
```

Comment:
This is inconsistent with plugin.yml (`[ list, content ]` vs `[list, content`) but I find it more readable, so I didn't change it.

Reply:
If you're gonna add a .yml stuff in the editorconfig please reformat the plugin.yml. Thanks!

Reply:
Done. More a question of "Is this desirable?" - the existing plugin.yml didn't match the .editorconfig at all, but we could create this section to match it rather than edit it.

Reply:
you want "spaces within braces" which doesn't match the current plugin.yml . either don't have that or reformat the plugin.yml . whatever of which is selected won't hurt the repo.

Reply:
I think spaces within braces are pointless, but I don't mind what we choose.